specs/multispec.yml

```diff
@@ -111,16 +112,29 @@ specs:
     "3.9":
       version: "3.9"
       short_ver: "39"
+      minimal: false
       base_img_version: "1"
       python_img_version: "1"
       scl: "rh-python39"
       module_stream: "python39:3.9"
       pkg_prefix: "python39"
       main_image: "registry.access.redhat.com/ubi8/python-39"
 
+    "3.9-minimal":
+      version: "3.9"
+      short_ver: "39"
+      minimal: true
+      base_img_version: "1"
+      python_img_version: "1"
+      scl: "rh-python39"
```

Comment:
Is this relevant? There's no rh-python39 collection at all. Maybe even remove it from the `3.9` config.